src/internal/easee/signalr.go

```diff
@@ -15,8 +14,10 @@ import (
 type SignalRManager interface {
 	root.Service
 
+	// Connected check if SignalR client is connected.
+	Connected() bool
```

Comment:
As I've mentioned in `connector.go` - in the future, this needs to check a connection state of a particular charger.